www/policy/PrivacyPolicy.html

```diff
@@ -100,12 +101,7 @@ <h2 id="s10">10. Legal mandates</h2>
 </p>
 
 
-<p>If you need to contact us in writing, address your mail to:</p>
-<p>
-CAcert Inc.<br />
-PO Box 66 <br />
-Oatley NSW 2223<br />
-Australia
+<p>If you need to contact us in writing, address your mail to the postal address of CAcert Inc. The current postal address of CAcert Inc. can be found on CAcert's web site.
```

Comment:
I find this very circular: isn't this the CAcert web site?  Could we just link to where the address is then?

Reply:
This is the text, voted by PolG